docs/asb/README.md

```diff
@@ -0,0 +1,134 @@
+# XMR to BTC Atomic Swap - Automated Swap Backend (ASB)
+
+## Quick Start ASB
+
+- Ensure that you have the Monero Wallet RPC running with `wallet-dir` and `disable-rpc-login`:
+  - `monero-wallet-rpc --stagenet --daemon-host STAGENET-NODE-URL --rpc-bind-port STAGENET-NODE-PORT --disable-rpc-login --wallet-dir PATH/TO/WALLET/DIR`
+- Run the ASB in terminal: `./asb start`
+- Follow the setup wizard in the terminal
+
+Public stagenet nodes:
+
+- `monero-stagenet.exan.tech`, `38081`
+- `stagenet.community.xmr.to`, `38081`
```

Comment:
Nit: This notation would make it clearer that 38081 is the port.

```suggestion
- `monero-stagenet.exan.tech:38081`
- `stagenet.community.xmr.to:38081`
```

Reply:
I specified it separately because of the command we use above where host and port are separate - happy to combine it here ;)